addons/arsenal/functions/fnc_buttonClearAll.sqf

```diff
@@ -0,0 +1,62 @@
+/*
+ * Author: Alganthe
+ * Clear the current container.
+ *
+ * Arguments:
+ * 0: Arsenal display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+*/
+#include "script_component.hpp"
+#include "..\defines.hpp"
+
+params ["_display"];
+
+// Clear container
+switch (GVAR(currentLeftPanel)) do {
+    case IDC_buttonUniform: {
+        {GVAR(center) removeItemFromUniform _x} foreach (uniformItems GVAR(center));
+        GVAR(currentItems) set [15, []];
+    };
+    case IDC_buttonVest: {
+        {GVAR(center) removeItemFromVest _x} foreach (vestItems GVAR(center));
```

Comment:
`clearItemCargo (vestContainer GVAR(center))` But that's ugly... And would probably also need clearMagazine and clearWeaponCargo.
But it would be 3 operations then. Instead of N operations for number of items in vest/uniform.
Also have to remember that vestItems returns list of all items with duplicates.

Reply:
I wasn't sure if clearItemCargo worked in this case, so I avoided using it.